home/components/PressableOpacity.tsx

```diff
@@ -1,24 +1,26 @@
 import { useExpoTheme, View } from 'expo-dev-client-components';
-import React, { PropsWithChildren } from 'react';
-import { PressableProps, Pressable, Platform, ViewStyle } from 'react-native';
+import React, { PropsWithChildren, ComponentProps } from 'react';
+import { PressableProps, Pressable, Platform } from 'react-native';
+
+type DCCViewProps = ComponentProps<typeof View>;
```

Comment:
```suggestion
import { PressableProps, Pressable, Platform, ViewProps  } from 'react-native';
```

RN exports `ViewProps`, so there is no need to define custom props via `ComponentProps` below.